src/components/FunctionsPanel/FunctionsPanel.js

```diff
@@ -108,7 +110,20 @@ const FunctionsPanel = ({
   const createFunction = deploy => {
     createNewFunction(project, functionsStore.newFunction).then(result => {
       if (deploy) {
-        return handleDeploy(functionsStore.newFunction)
+        const data = {
+          ...functionsStore.newFunction
+        }
+
+        if (
+          data.spec.build.commands.length > 0 &&
```

Comment:
I think this condition is unnecessary, because for an empty array: `[].includes(value)` will return `false` anyway.